vue-app/src/plugins/Web3/constants/tokens.ts

```diff
@@ -0,0 +1,29 @@
+export interface Token {
+  logo: string
+}
+
+// Can lookup token information by passing
+// TOKEN_INFO[nativeTokenSymbol.toUpperCase()]
+export const TOKEN_INFO: { [key: string]: Token } = {
+  ETH: {
+    logo: 'eth.svg',
+  },
+  DAI: {
+    logo: 'dai.svg',
+  },
+  WXDAI: {
+    logo: 'wxdai.svg',
+  },
+  USDC: {
+    logo: 'usdc.svg',
+  },
+  WETH: {
+    logo: 'weth.svg',
+  },
+  AOE: {
+    logo: 'aoe.svg',
+  },
+  MATIC: {
+    logo: 'matic.svg',
+  },
+}
```

Comment:
I was wondering whether we want these constants in a new `/constants` folder in the root of vue-app. I'm starting to think that this is more app related than web3 plugin related.

Its also true that the `chain.ts` file is kind of in the middle. So, for simplicity we can keep this all in here at the moment.